components/raftstore/src/store/worker/pd.rs

```diff
@@ -2381,24 +2394,28 @@ fn new_split_region_request(
     new_region_id: u64,
     peer_ids: Vec<u64>,
     right_derive: bool,
+    skip_size_check: bool,
 ) -> AdminRequest {
     let mut req = AdminRequest::default();
     req.set_cmd_type(AdminCmdType::Split);
     req.mut_split().set_split_key(split_key);
     req.mut_split().set_new_region_id(new_region_id);
     req.mut_split().set_new_peer_ids(peer_ids);
     req.mut_split().set_right_derive(right_derive);
+    req.mut_split().set_skip_size_check(skip_size_check);
     req
 }
 
 fn new_batch_split_region_request(
     split_keys: Vec<Vec<u8>>,
     ids: Vec<pdpb::SplitId>,
     right_derive: bool,
+    skip_size_check: bool,
 ) -> AdminRequest {
     let mut req = AdminRequest::default();
     req.set_cmd_type(AdminCmdType::BatchSplit);
     req.mut_splits().set_right_derive(right_derive);
+    req.mut_splits().set_skip_size_check(skip_size_check);
```

Comment:
What does  `skip_size_check`?

Reply:
if `skip_size_check` is true, the new splitted region's initial size is zero not the half size of the source region.

Reply:
How about `inherit_size`?

Reply:
has change as amortize_source_region_size.